CMakeLists.txt

```diff
@@ -270,7 +272,8 @@ INCLUDE(${KOKKOS_SRC_PATH}/cmake/kokkos_install.cmake)
 # executables also need nvcc_wrapper.  Thus, we need to install it.
 # If the argument of DESTINATION is a relative path, CMake computes it
 # as relative to ${CMAKE_INSTALL_PATH}.
-INSTALL(PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/bin/nvcc_wrapper DESTINATION ${CMAKE_INSTALL_BINDIR})
+KOKKOS_INSTALL_FILES()
```

Comment:
is this the right name? I mean it doesn't install all Kokkos files or? It only installs some subset?

Reply:
renamed to KOKKOS_INSTALL_ADDITIONAL_FILES